Hack Day: Tyk Self Managed : Update Quick Start To Offer Free Trial

dcs3spp opened the pull request:
[Preview Self Managed Free Trial](https://deploy-preview-3175--tyk-docs.netlify.app/docs/nightly/getting-started/quick-start/)
[Preview Tyk On Premises](https://deploy-preview-3175--tyk-docs.netlify.app/docs/nightly/tyk-on-premises/)

sedkis reviewed (approved):
Lovely:

https://deploy-preview-3175--tyk-docs.netlify.app/docs/nightly/getting-started/quick-start/

Thanks!

dcs3spp commented:
Hi @sedkis many thanks for reviewing. I've updated the PR so that CTA buttons for free trial links directly to Self Managed signup page and added the What's Included section. When you get chance would you be able to re-review and I will then merge and release

dcs3spp commented:
/release to release-5.1
